src/dlls/mscoree/mscorwks_unixexports.src

```diff
@@ -4,6 +4,7 @@ CopyFileW
 CoreDllMain
 CoTaskMemAlloc
 CoTaskMemFree
+CreateDelegate
```

Comment:
We should group methods in this file based on the purpose: the public hosting methods that you are adding, Win32 APIs used by mscorlib, exception handling helpers used by LLILC.